raycicmd/config.go

```diff
@@ -119,6 +119,12 @@ type config struct {
 	// Optional.
 	AllowTriggerStep bool `yaml:"allow_trigger_step"`
 
+	// RestrictedConcurrencyGroup is the list of concurrency group name prefixes that are
```

Comment:
rename to `AllowedConcurrencyGroupPrefixes` ?

`Restricted` feels that it is not permitted to use..